src/fr/composants/base.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Looks like this file is a duplicate of `src/fr/composants/Case-a-cocher/base.md` - was that intentional?